src/js/main.js

```diff
@@ -804,6 +806,95 @@ swiper.addEventListener(
 	supportsPassive ? { passive: true } : false
 );
 
+const weatherSection = document.querySelector(".weather-section");
+
+async function getWeather() {
+	const response = await fetch(
+		"http://api.openweathermap.org/data/2.5/weather?q=Salto,uy?&units=metric&APPID=b0142355ff1172118bcf173d1bd9f022"
```

Comment:
Shouldn't this be Arapey?

Reply:
Is it possible to do a `https` request?

Reply:
Well I tried to look for "Arapey" or "Termas del Arapey", but they can't be found. Salto is the nearest location that could be found though.